src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -1,11 +1,41 @@
 package com.kodilla.ecommercee.controller;
 
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.kodilla.ecommercee.controller.exeptions.CartNotFoundException;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
-@CrossOrigin("*")
 @RequestMapping(value = "v1/cart/",produces = "application/json")
+@CrossOrigin("*")
+
 public class CartController {
+
```

Comment:
Ze specyfikacji projektu wynika, że powinny być metody, które powodują:
a) utworzenie pustego koszyka (bym to widział, że tworzysz nowy pusty koszyk z ArrayListą Productów)
b) pobranie elementów z pustego koszyka (get jakiś produkt z listy w koszyku)
c) dodanie elementów (produktów) do koszyka (add element do listy w koszyku)
d) usunięcie konkretnego produktu z koszyka (remove element z listy w koszyku)
e) utworzenie zamówienia na podstawie koszyka (create order z produktami z koszyka) 

To tylko moje przemyślenia, ale możesz na tej podstawie coś pokombinować. :-) Może ktoś coś jeszcze zasugeruje?

Reply:
Wielkie dzięki za pomoc :) Wprowadziłem korekty. Mógłbyś rzucić okiem

Reply:
Ok wprowadziłem korekty i przeszły testy. Jeżeli będziesz miał chwilę to zerknij proszę na CartController czy ja czegoś tam nie pomieszałem. Starałem się uprościć te klasy. Z góry dzięki